### ISSUE TITLE
Add space between TTY & number

### ISSUE THREAD

**Issue by AGarcia-Clarity** [department-of-veterans-affairs/va.gov-team#34447]
## Issue Description
A bug exists in the SubmitSignInForm component in which the "TTY" and its corresponding phone number does not contain a space between them.  To increase accessibility and readability a space should be added between them to ease user experience.

---
## Tasks
- [x] Add a space between "TTY" and the phone number 711

## Acceptance Criteria
- [x] I can see a space between the TTY & number
- [ ] The content reads as follows: "TTY: 711"
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
